<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1131</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

color:rgb(24, 50, 109);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
  



    border: 3px solid white;



}</string>
      </property>
      <property name="title">
       <string>Formulaire</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>CIN:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Prenom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Numero:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
       </property>
       <property name="text">
        <string>Adresse:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajout">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}

</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>310</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}
</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>360</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}
</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="le_adresse_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tunis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BenArous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ariana</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLineEdit" name="chercher_num">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	font: 63 10pt &quot;Segoe UI Semibold&quot;;
 

color:rgb(24, 50, 109);
}
</string>
      </property>
      <property name="text">
       <string>Tri par :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}
</string>
      </property>
      <property name="text">
       <string>SMS</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_client">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>211</y>
        <width>711</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>inserer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color : rgb(24, 50, 109);
}</string>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_cin">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}

</string>
      </property>
      <property name="text">
       <string>Cin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf_3">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}

</string>
      </property>
      <property name="text">
       <string>pdf </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}</string>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}

</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}

</string>
      </property>
      <property name="text">
       <string>Numero</string>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>label_7</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>chercher_num</zorder>
     <zorder>tab_client</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>label_6</zorder>
     <zorder>tri_cin</zorder>
     <zorder>pdf_3</zorder>
     <zorder>pushButton</zorder>
     <zorder>tri_2</zorder>
     <zorder>tri</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="imagee">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>401</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="photo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius:15px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-right :1px solid #aaaaaa;
border-bottom :1px solid #aaaaaa;
}
QPushButton:enabled {
background-color: #E1F4FF;
color: #0c2f70;
}
QPushButton:pressed {
background-color:blue;
color: white;
}

QPushButton:hover:!pressed {

background-color : #0d2f72;
color: #fffffe;

}
QPushButton:disabled{
boackground-color:#aaaaaa;
color :ffffff;
}

</string>
      </property>
      <property name="text">
       <string>upload</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
